pkg/utils/environment.go

```diff
@@ -38,6 +38,20 @@ func (engineDetails *EngineDetails) SetEngineUID(clients ClientSets) error {
 
 //SetENV sets ENV values in experimentDetails struct.
 func (expDetails *ExperimentDetails) SetENV(engineDetails EngineDetails, clients ClientSets) error {
+	// Setting envs from engine fields other than env
+	expDetails.setEnv("CHAOSENGINE", engineDetails.Name).
```

Comment:
nit: Do we really need builder for setting ENV's. I would suggest a slice of ENV variables, as in `v1.EnvVar`, and then we don't have to do same function call again and again.

Reply:
- Added function to exclude all the envs who contains an empty value
-  as the number of envs increases the slice size increases which can fail bch guidelines(4x lines)